### DISCUSSION TITLE
Consider making SSW Rewards a web app

### DISCUSSION THREAD

**Discussion by Calinator444** [SSWConsulting/SSW.Rewards.Mobile#498] [Ideas]
As someone who initially heard about SSW rewards before starting my internship with SSW I think I might have some useful feedback coming from an outsider perspective.

Here are the main reason I didn't install SSW rewards:
❌The app is only relevant when I'm visiting an event hosted by SSW
❌As a user I may be installing an app which I'll only ever use once and forget about

💡suggestion: might I recommend making the app a web app instead? Similar to for instance [Retrospected](https://app.retrospected.com/game/civ_jzda4)? That way the users can simply fill out the form with the their email and phone number and be added to a mailing list when the prizes are drawn.

So for instance a new use case might look like this:

1. Scan a QR code taking them straight to SSW rewards web page for that event
2. The user fills out any relevant information/feedback for that event
3. The user enters their email and phone number into the app, adding them to the mailing list when prizes are drawn
4. If the user wins a prize they receive a text message/email telling them when and where they can collect their prize

Please let me know what you think of this idea in the comments.

**Comment by matt-goldman** [comment]
Hi Caleb

Thanks for this suggestion. As much as any of the functionality delivered, SSW Rewards serves to act as a showcase mobile app for SSW. Its purpose is to demonstrate our capability developing mobile apps as much as it is to deliver the functionality provided by the app. For that reason, it's important that we keep it as a mobile app.

There are also other reasons why many users prefer a mobile app, and also functionality on the roadmap that would be difficult (or impossible) to do with a web app.

However, this could change in the future. If we develop more mobile apps we may revisit this decision for SSW Rewards.

**Comment by adamcogan** [comment]
Guys
I understand the motivation for Caleb's suggestion, because there are always some % lost as some are reluctant to install a mobile app.
However I am 100% with Goldie's response and want it to remain as a mobile app.
Note: If we were in China I would develop a WeChat mini-app as that is the perfect in between.

Hey @sethdaily 
1. Could you work with Penny and reply with the booth signage we use at events.
It should mention that the mobile app allows you to earn points with an AI quiz, you can redeem merch, you will get a notification to attend the big draw at 4pm etc.
-a

**Reply by marinagerber** [reply]
I’m always guided by the 'why' behind any decision - understanding the reasoning is what drives my response.

When I understand the purpose and objectives of the Rewards app, it makes sense to me as a solution.

If the main goal of the SSW Rewards program is marketing-focused, then developing a web app feels like the more effective and strategic choice."

**Comment by AlvinShenSSW** [comment]
I understand Caleb and Goldie’s different perspectives. I want to share my thoughts as I am responsible for having SSW's and our clients software run in China aka Chinafy.

In China most people are reluctant to download full apps, especially those they do not use frequently. In China, it’s common for each app to have a full app and several mini-app versions, such as WeChat Mini Programs and Alipay Mini Apps. 

For an app like SSW Rewards, I would like to use it at the Fire User Group in Hangzhou. Only 1/2 could install the iphone app and the other 1/2 cannot install the Android app and would need an alternative. All would prefer a Mini-app but could live with a web app.

A Mini-app allows them to, scan it, use it, and throw it.  

If the SSW Rewards wants to be used in China, supporting the Chinese language and WeChat Mini-app is essential. (In China, most people tend to use Mini-app, and the Google Play Store does not even exist in China, so Android users find it hard to get the whole app easily)
